test/dlc_tests/container_tests/bin/testSmdebug

```diff
@@ -41,6 +41,7 @@ git pull
 echo "Creating mock test folder."
 mkdir -p mock
 cp -r tests mock/
+cp -r examples mock/
```

Comment:
why do we need this?

Reply:
There are tests in the `sagemaker-debugger` repo which trigger scripts in the `examples` folder.

By moving the test files into `mock/`, and changing folders into the `mock/` folder, I changed the import path available to the tests. So I moved the `examples/` folder into `mock/` as well.

The reason for moving these tests was to prevent our tests from accidentally importing source code from the `smdebug/` folder, instead of using the `smdebug` module installed in `site-packages`.